pkgs/top-level/all-packages.nix

```diff
@@ -11018,6 +11018,8 @@ in
 
     tp_smapi = callPackage ../os-specific/linux/tp_smapi { };
 
+    tpacpi-bat = callPackage ../os-specific/linux/tpacpi-bat { };
```

Comment:
There is no need for this packages to be in `linuxPackages` as it's just a script that doesn't depend on any specific kernel for building. Only out-of-tree kernel modules are typically in `linuxPackages`.